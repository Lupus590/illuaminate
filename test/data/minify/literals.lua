local x

x = 0
x = 0., .0, 0.0
x = 0. .. .0
x = - - 2
x = 2 and 2
do local x = 0 end

x = "hello"
x = 'hello'
x = [[hello]]
x = x[ [[hello]] ]
