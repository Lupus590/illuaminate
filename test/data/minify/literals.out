local e e=0 e=0.,.0,0.0 e=0. .. .0 e=- -2 e=2 and 2 do local e=0 end
e="hello"e='hello'e=[[hello]]e=e[
[[hello]]]